src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java

```diff
@@ -536,20 +536,22 @@ private static StringBuilder getServedClaimsReport(ListingData listingData) {
 
     private static void addEntriesByServingDay(int dayNumber, String listBlockOpener,
                                                StringBuilder reportContent, ListingData listingData) {
-
         var itemsList = listingData.getLocalReportsDetail().get(0);
         var claimServedTypeItems = itemsList.getValue().getClaimServedItems()
-                .stream()
-                .filter(item -> Integer.parseInt(item.getValue().getReportedNumberOfDays()) == dayNumber)
+                .stream().filter(item -> Integer.parseInt(item.getValue().getReportedNumberOfDays()) == dayNumber)
                 .collect(Collectors.toList());
         int claimServedTypeItemsCount = claimServedTypeItems.size();
         var claimServedTypeItemsListSize = String.valueOf(claimServedTypeItems.size());
 
+        if (dayNumber >= 5) {
```

Comment:
Is this not supposed to be >=6?

Reply:
As 5 days is in it's own section, surely it should be any that is equal or more than 6?

Reply:
No it is 5 Harpreet.

Reply:
The days count is a bit tricky. But the logic is working as Dan requested.

Reply:
Looked in the code to help my understanding, looks good to me now